language/vm/src/proptest_types.rs

```diff
@@ -24,7 +24,7 @@ mod signature;
 use crate::proptest_types::functions::{FnHandleMaterializeState, FunctionHandleGen};
 use functions::{FnDefnMaterializeState, FunctionDefinitionGen};
 use signature::{KindGen, SignatureTokenGen};
-use std::collections::{BTreeSet, HashSet};
+use std::collections::{BTreeSet, HashMap, HashSet};
```

Comment:
`TreeMap` to avoid accidental nondeterminism from iterating over this?

Reply:
I may change it at some point just for "safety".
The point is that right now those are literally used for hash and not iteration at all. Basically they are used to keep track of some binary value to its index...